Renderer.js

```diff
@@ -264,7 +264,7 @@
 		if (newValue == null || (element.type === 'number' && isNaN(newValue))) {
 			newValue = ''
 		}
-		if (element[this.name] != newValue) {
+		if ((element.type === 'number' && element[this.name] !== newValue) || element[this.name] != newValue) {
```

Comment:
`'' == 0` -> `true`
not sure if a more sophisticated type/coercion negotiation is warranted here...